src/app/_utils/createMetadata.ts

```diff
@@ -43,16 +43,16 @@ export function createMetadata({
 
   return {
     title: overrideDefaultTitle
-      ? { absolute: parsedEnrichedSEO.title }
-      : parsedEnrichedSEO.title,
+      ? { absolute: parsedEnrichedSEO.title as string }
+      : (parsedEnrichedSEO.title as string),
     description: parsedEnrichedSEO.description,
     openGraph: {
       title: parsedEnrichedSEO['open-graph']?.title,
       description: parsedEnrichedSEO.description,
       images: parsedEnrichedSEO['open-graph']?.image,
     },
     twitter: {
-      card: parsedEnrichedSEO.twitter?.card,
+      card: parsedEnrichedSEO.twitter?.card ?? 'summary',
```

Comment:
I think Typescript registers from this` card: (seo.twitter?.card || 'summary')` (line 35)  a union type `("summary" | "summary_large_image" | "player" | undefined)`, and it's giving an error that it can't be undefined. You think we could move the defaults together in the return object?

Reply:
I'm not sure how this is possible but the TS error already exists in `main` - I remember pairing on this together, and there are many options that we need to test, so I wouldn't just move this to the return object. We need to have a closer look at this.